test/chemical/test_residue.py

```diff
@@ -15,12 +15,12 @@ def test_variable_checks():
     assert len(res.AA_LIST) == 21
     assert all_caps(res.AA_LIST)
 
-    assert len(res.THREE_LETTER_AA_MAPPER) == 29
```

Comment:
It should be 29 after those added names are back

Reply:
Updated.